const.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package marathon
 
 const (
+	DEBUG_LEVEL        = 10
```

Comment:
`DebugLevel`?
As I know constants follow the same rule as usual variables: https://github.com/golang/glog/blob/master/glog.go#L100